src/revpi-factory-reset

```diff
@@ -123,7 +132,7 @@ if [[ "$ovl" =~ ^($witheth1_devicetype_list)$ ]] ; then
 		echo "HDMI disabled. Use 'sudo systemctl disable hdmi-disable' and reboot to reenable HDMI."
 	fi
 fi
-if [ "$kernel" = 44 ] ; then
+if [ "$kernel" = 44 && ! hats-detection ] ; then
```

Comment:
We will never provide a 4.4 kernel for any device with a hat-eeprom

Reply:
but theoretically, the user running jessie (4.4kernel) with updated firmware can reach this